### PR TITLE
Added a logger for datetime formatter.

### PR DESCRIPTION
**Pull request by mehrdadhalali**

### CODE REVIEW

File: web_scraping/gog_scraper/scrape_gog_game.py [hunk 1]
```diff
@@ -2,9 +2,12 @@
 
 from datetime import datetime
 from urllib.request import urlopen
+import logging
```
Comment (Leon-JS02) [inline web_scraping/gog_scraper/scrape_gog_game.py:5]: Great idea mehrdad

File: web_scraping/gog_scraper/scrape_gog_game.py [hunk 2]
```diff
@@ -107,8 +111,11 @@ def get_game_data_from_url(game_url: str) -> dict:
                                           item_class="details__content table__row-content",
                                           are_links=False)
     operating_systems = format_os(operating_systems)
-    release_date = datetime.strptime(release_date[3:13],
-                                     "%Y-%m-%d")
+    try:
```
Comment (Leon-JS02) [inline web_scraping/gog_scraper/scrape_gog_game.py:114]: Date times can cause difficult bugs sometimes, nice idea.

**Review by Leon-JS02 — APPROVED** [review]
Nice changes.

File: web_scraping/gog_scraper/scrape_gog_game.py (hunk 1 above)
Comment (paulauus) [reply web_scraping/gog_scraper/scrape_gog_game.py:5]: Next time will know to do this from the start :')

**Review by paulauus — APPROVED** [review]

**Pull request merged**